### PR TITLE
Adds Fennix mob

### PR DESCRIPTION
**Pull request by Ketrai**
Adds a smol fennix mob. 
- Protects other fennecs.

Original spritework done by me. Will add vorebelly sprites, and resting sprites later, and maybe some little extra code for neat tricks.

Will add to a fennec cargo crate too.

### CODE REVIEW

File: vorestation.dme [hunk 1]
```diff
@@ -2214,6 +2214,7 @@
 #include "code\modules\mob\living\simple_animal\vore\dino.dm"
 #include "code\modules\mob\living\simple_animal\vore\dragon.dm"
 #include "code\modules\mob\living\simple_animal\vore\fennec.dm"
+#include "code\modules\mob\living\simple_animal\vore\Fennix.dm"
```
Comment (Arokha) [inline vorestation.dme]: Lowercase filename, since the filename you used doesn't have a caps F

**Comment by Ketrai** [comment]
Just edited fennix to do 1-2 dmg on retaliation, because as with fennecs, we don't really want to encourage people to kill them. So really, it's more cute than dangerous.

SPRITES PREVIEW : https://imgur.com/a/U1TXWbY

**Comment by Ketrai** [comment]
Going to go sleep for now.

**Comment by SigholtStarsong** [comment]
I will only approve of these being added if I can put collars on them to make them tame.

**Comment by Ketrai** [comment]
So uh, any changes needed? Or can this be merged now? ._.

**Pull request merged**